network_bugs_overview

```diff
@@ -349,8 +352,25 @@ def process_jira_bugs(bugs, developers):
                           bug, e, severity))
                 severity = 'unspecified'
 
+        # an assignee in jira bugs is very often a developer's official email address, but
+        # it might also be just a username. On BZ aliases are the primary choice.
         if assignee not in developers:
-            continue
+            if "@" in assignee:
+                # skip this bug
+                continue
+            else:
+                # try with full email address if assignee is not in that form already
+                tmp_assignee = "{}@redhat.com".format(assignee)
+                if tmp_assignee in developers:
```

Comment:
LGTM but not sure if works, I am wondering if we could simplify the logic:
```
 if tmp_assignee in developers  OR RH_DEVELOPERS_ALIASES.get(assignee)
     assignee = tmp_assignee
else
    continue
```

Reply:
hsould be ok, @dougsland the .get() is safe to call if key does not exist.